### PR TITLE
Test/KLBP-252: Refactoring folders structure.

### PR DESCRIPTION
**Pull request by iuliudascalu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +46,7 @@ out new developers:
 components like:
     * [Alerts](src/components/Alerts)
     * [FormGroup](src/components/FormGroup) - contains basic input components based on react hook form
-    * [Modal](src/components/Modal)
+    * [Modal](src/components/Journeys/{companyType}/Modal)
```
Comment (mariuspop86) [inline README.md:49]: Please revert this change

File: src/__tests__/components/Modal/ModalContainer.test.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import { render, screen } from '@testing-library/react'
 import '@testing-library/jest-dom/extend-expect'
-import ModalContainer from '../../../components/Modal/ModalContainer'
+import ModalContainer from '../../../components/Journeys/IndividualCompany/Modal/ModalContainer'
```
Comment (mariuspop86) [inline src/__tests__/components/Modal/ModalContainer.test.tsx:4]: Please revert this change

File: src/components/Journeys/IndividualCompany/Step2/IndividualCompanyStep2Content.tsx [hunk 3]
```diff
@@ -1,15 +1,20 @@
...
-import useStep2Hook from '../../hooks/useStep2Hook'
-import DisclaimerSection from './DisclaimerSection'
-import { isJourneyRevisit } from '../../services/utils'
+import StepNavButtons from '../../../StepNavButtons/StepNavButtons'
+import useStep2Hook from '../../../../hooks/useStep2Hook'
+import DisclaimerSection from '../../Common/DisclaimerSection'
+import { isJourneyRevisit } from '../../../../services/utils'
 
-function Step2Content(): ReactElement {
+function IndividualCompanyStep2Content(): ReactElement {
```
Comment (mariuspop86) [inline src/components/Journeys/IndividualCompany/Step2/IndividualCompanyStep2Content.tsx:9]: Please leave the old name

File: src/components/Journeys/IndividualCompany/LegalRepresentativeSection/LegalRepresentativeSection.tsx [hunk 4]
```diff
@@ -1,10 +1,10 @@
 import React, { ReactElement } from 'react'
 import { Button } from 'kyb-core'
-import { DocumentLabelById, DocumentTypeOptions } from '../step2/Options'
+import { DocumentLabelById, DocumentTypeOptions } from '../Step2/Options'
```
Comment (ioan-iacob-netea) [inline src/components/Journeys/IndividualCompany/LegalRepresentativeSection/LegalRepresentativeSection.tsx:3]: :+1:

File: src/components/Steps/Step1Content.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import { IStore } from '../../interfaces/Store'
+import { CLIENT_TYPOLOGY } from '../../enums/journeys'
+import IndividualCompanyStep1Content from '../Journeys/IndividualCompany/Step1/IndividualCompanyStep1Content'
```
Comment (mariuspop86) [inline src/components/Steps/Step1Content.tsx:5]: .

File: src/components/Steps/Step1Content.tsx [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  const { typologie_client } = useSelector(
+    (state: IStore) => state.folderReducer.folder
+  )
+  const { INDIVIDUAL_COMPANY } = CLIENT_TYPOLOGY
+
+  switch (typologie_client) {
+    case INDIVIDUAL_COMPANY:
+      return <IndividualCompanyStep1Content />
+    default:
+      return <div>private company journey step1 content</div>
```
Comment (mariuspop86) [inline src/components/Steps/Step1Content.tsx:17]: For default please redirect to 404

File: src/components/Journeys/IndividualCompany/Step2/Options.ts [hunk 7]
```diff
@@ -1,7 +1,7 @@
-import { IOption } from '../../interfaces/FormFields'
-import { RL_PERSON_TYPE_ID } from '../../enums/journeys'
-import { IFileState } from '../../interfaces/Upload'
-import { IPersonData } from '../../interfaces/Step2'
+import { IOption } from '../../../../interfaces/FormFields'
```
Comment (mariuspop86) [inline src/components/Journeys/IndividualCompany/Step2/Options.ts:1]: Please move all of this into the enum folder

**Review by ioan-iacob-netea — CHANGES_REQUESTED** [review]

**Review by ioan-iacob-netea — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (iuliudascalu) [reply README.md]: Done.

File: src/__tests__/components/Modal/ModalContainer.test.tsx (hunk 2 above)
Comment (iuliudascalu) [reply src/__tests__/components/Modal/ModalContainer.test.tsx]: Done.

File: src/components/Journeys/IndividualCompany/Step1/IndividualCompanyStep1Content.tsx [hunk 8]
```diff
@@ -1,9 +1,9 @@
 import React, { ReactElement } from 'react'
-import { useStep1FormHook } from '../../hooks/useStep1FormHook'
+import { useStep1FormHook } from '../../../../hooks/useStep1FormHook'
 import Step1InfoSection from './Step1InfoSection'
 import { Step1Form } from './Step1Form'
 
-function Step1Content(): ReactElement {
+function IndividualCompanyStep1Content(): ReactElement {
```
[the remark this answers is not in the document]
Comment (iuliudascalu) [reply src/components/Journeys/IndividualCompany/Step1/IndividualCompanyStep1Content.tsx:6]: Done.

File: src/components/Journeys/IndividualCompany/Step2/IndividualCompanyStep2Content.tsx (hunk 3 above)
Comment (iuliudascalu) [reply src/components/Journeys/IndividualCompany/Step2/IndividualCompanyStep2Content.tsx]: Done.

File: src/components/Steps/Step1Content.tsx (hunk 6 above)
Comment (iuliudascalu) [reply src/components/Steps/Step1Content.tsx]: Done.

File: src/components/Journeys/IndividualCompany/Step2/Options.ts (hunk 7 above)
Comment (iuliudascalu) [reply src/components/Journeys/IndividualCompany/Step2/Options.ts:1]: Done.

File: src/components/Steps/Step1Content.tsx [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+  )
+  const { INDIVIDUAL_COMPANY } = CLIENT_TYPOLOGY
+  const history = useHistory()
+
+  switch (typologie_client) {
+    case INDIVIDUAL_COMPANY:
+      return <IndividualCompanyStep1Content />
+    default:
+      history.push(ROUTES.notFound)
+      return <></>
```
Comment (mariuspop86) [inline src/components/Steps/Step1Content.tsx:21]: Remove `history.push(ROUTES.notFound)`
```suggestion
      return <Redirect to={ROUTES.notFound} />
```
Try not to mix js with jsx

Comment (iuliudascalu) [reply src/components/Steps/Step1Content.tsx]: Done.

**Review by mariuspop86 — APPROVED** [review]
